src/store/slices/customerSlice.ts

```diff
@@ -0,0 +1,57 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import type { RootState } from '../../store';
+import { API } from '../../api/API';
+import { getApiRoot } from '../../api/lib/Client';
+import { type Customer } from '@commercetools/platform-sdk';
+
+export interface Credentials {
+  email: string;
+  password: string;
+  // setOpen: (val: boolean) => void;
```

Comment:
If you won't use it in future, it is better to remove commented code. Otherwise add todo.